README.md

```diff
@@ -4,44 +4,41 @@
 
 RESTEasy is a JBoss.org project aimed at providing productivity frameworks for developing client and server RESTful applications and services in Java.  It is mainly a JAX-RS implementation but you'll find some other experimental code in the repository.
 
-The project page can be found at https://resteasy.jboss.org
+The project page can be found at [https://resteasy.github.io/](https://resteasy.github.io/).
 
 ## JAX-RS
-RESTEasy is a JBoss project that provides various frameworks to help you build RESTful Web Services and RESTful Java applications. It is a portable implementation of the JAX-RS specification. JAX-RS is a JCP specification that provides a Java API for RESTful Web Services over the HTTP protocol.
 
-Read the entire specification [here](https://github.com/jax-rs)
+RESTEasy is a JBoss project that provides various frameworks to help you build RESTful Web Services and RESTful Java applications. It is a portable implementation of the JAX-RS specification. JAX-RS is a JCP specification that provides a Java API for RESTful Web Services over the HTTP protocol.
 
-The RESTEasy code is in the github repository, (https://github.com/resteasy/Resteasy.git).
+Read the entire specification [here](https://github.com/eclipse-ee4j/jaxrs-api).
 
 ## Getting started with RESTEasy
+
 - Read a [book](https://resteasy.github.io/books.html)
 - Check out the [examples](https://github.com/resteasy/resteasy-examples) in the repository.
 - Read the [documentation](https://resteasy.github.io/docs).
+- The new blog is [here](https://github.com/resteasy/resteasy-blog).
 
 ## Documentation
 
-To read the documentation you can [read it online](https://resteasy.github.io/docs). This is generated automatically from the [maven build](https://github.com/resteasy/Resteasy/tree/master/jaxrs/docbook)
-
-A wiki exists with historical documents. You can find it [here](https://developer.jboss.org/wiki/ResteasyWIKI)
+To read the documentation you can [read it online](https://resteasy.github.io/docs).
 
 ## Issues
+
 Issues are kept in [JIRA](https://issues.jboss.org/browse/RESTEASY).
 
 ## Build
+
 After pulling down a clone of the RESTEasy repository run
 
-	mvn install
+```bash
+$ mvn install
+```
 
 Currently it can be built using JDK 1.8 and 11.
-If you don't have the JBoss Nexus repository configured in your own local Maven settings, you might need to add enable the 'jboss-repository' profile:
-
-       mvn -Pjboss-repository install
-
-
-### Requirements
-- Java Cryptography Extension [JCE](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html)
```

Comment:
Why are you removing these?

Reply:
I thought we no longer need these to build, just pure `mvn clean install` can build the project, am I missing anything?

Reply:
The -Pjboss-repository is still needed in some cases (the text says "might need to add", which is true). The JCE, you're right, it's not a strict requirement

Reply:
@asoldano okay!

Reply:
I'll double check it whether it can be removed in my local env.